### ISSUE TITLE
[FIX] Clear notifications after opening the app 

### ISSUE THREAD

**Issue by eliezer-of** [RocketChat/Rocket.Chat.ReactNative#690]
when a message from a favorite chat is received and it’s open directly in Rocket.Chat
app, it would be interesting to removed in notification.

**Comment by diegolmello** [comment]
@eliezer-of It's probably not in favorite chats only, but it would be nice.
The rule is to check if the messages were read anywhere (web, desktop, other apps).

**Comment by diegolmello** [comment]
@eliezer-of Are you aware of any apps that do something like this?

**Comment by eliezer-of** [comment]
> @eliezer-of It's probably not in favorite chats only, but it would be nice.
> The rule is to check if the messages were read anywhere (web, desktop, other apps).

@diegolmello yea! not just on favorite chats, I mean any notification received in the app.
But I specifically refer to the app (Reactive.Native) after reopening it and changing AppState to "onResume", it should clear all notifications. (i tested this on android)


> @eliezer-of Are you aware of any apps that do something like this?

not exactly, because I opened the same issue in the Android repository. And it was fixed as I said before
